src/core/config/defaults.go

```diff
@@ -103,7 +103,8 @@ const (
 	DynamicConfigFileAbsFreeBsdPath = "/var/db/nginx-agent/agent-dynamic.conf"
 	ConfigFileName                  = "nginx-agent.conf"
 	ConfigFileType                  = "yaml"
-	EnvPrefix                       = "nms"
+	OldEnvPrefix                    = "nms"
+	NewEnvPrefix                    = "nginx_agent"
```

Comment:
nit: I think `OldEnvPrefix` should rather be called `LegacyEnvPrefix` and `NewEnvPrefix` just `EnvPrefix`. That way we could eventually get rid of `LegacyEnvPrefix` and we'd be left with `EnvPrefix`.